### DISCUSSION TITLE
Can I use it for desktop App?

### DISCUSSION THREAD

**Discussion by Shaheerimam** [dlutton/flutter_tts#276] [Q&A]
Please tell me if there any way to use tts in **_Desktop App._**

**Comment by dlutton** [comment]
for macos yes, however I don't have it ported for windows/linux yet.

**Comment by Shaheerimam** [comment]
Will you update it soon for Desktop?

**Reply by dlutton** [reply]
when I get some time yes

**Comment by Shaheerimam** [comment]
Great, thanks!

**Reply by dlutton** [reply]
@Shaheerimam windows support has been added.
